### DISCUSSION TITLE
How to add custom card for all kind of TV remote which can control over IP,RS232 and IR

### DISCUSSION THREAD

**Discussion by kapilt007** [dwainscheeren/dwains-lovelace-dashboard#625] [Dwains Dashboard V3 Alpha]
Dear Sir,

I am not able to add TV remote custom card in Dwains dashboard,

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
